flink-libraries/flink-table/src/main/scala/org/apache/flink/table/runtime/functions/ScalarFunctions.scala

```diff
@@ -104,9 +103,84 @@ object ScalarFunctions {
     }
     if (base <= 1.0) {
       throw new IllegalArgumentException(s"base of 'log(base, x)' must be > 1, but base = $base")
-    }
-    else {
+    } else {
       Math.log(x) / Math.log(base)
     }
   }
+
+  /**
+    * Returns the string str, left-padded with the string pad to a length of len characters.
+    * If str is longer than len, the return value is shortened to len characters.
+    */
+  def lpad(base: String, len: Integer, pad: String): String = {
+    if (base == null || len == null || pad == null) {
```

Comment:
`addSqlFunctionMethod` already checks for null in the arguments so we can use primitive integer here and remove this checks.